src/functions/sendCandidateResults/framework/request-scheduler.ts

```diff
@@ -107,6 +114,11 @@ export class RequestScheduler implements IRequestScheduler {
     });
   }
 
+  protected getNumberFromEnv(envvarName: string): number | null {
```

Comment:
This exists in config adapter

Reply:
Yup, will move things from config adapter